### PR TITLE
ui: Instance detail redesign

### PR DESCRIPTION
**Pull request by kaxcode**
These changes do not include the new proxy tab, but they do remove proxy tests that no longer have valid scenarios. 

[Preview of the Instance Detail page redesign](https://deploy-preview-7683--consul-ui-staging.netlify.app/ui/ar_west-5/services/service-0/instances/node-0/service-0-with-id/health-checks)

- Remove Proxy link and add ExternalSource to instance detail page header
- Create HealthChecks tab with route and styling
- Fix up tests to fit redesign of Service Instances Detail page

### CODE REVIEW

**Comment by kaxcode** [comment]
> Is there anyway we can split the breadcrumbs changes off into a separate PR? The changes to breadcrumbs will affect the entire UI throughout KV, Intentions, ACLs etc etc not just the instance detail page. If we do that we can just concentrate on the Instance detail redesign here, and the breadcrumbs changes in the separate PR.
> 
> P.S. Actually I've just looked at your commits, if this a hassle to don't worry we can go through it all as one PR, lemme know when you are about.

- Breadcrumbs styling is now in PR #7687

File: ui-v2/app/styles/components/app-view/layout.scss [hunk 1]
```diff
@@ -20,6 +20,9 @@
 %app-view-header dt {
   font-weight: bold;
 }
+%app-view-header dd > a {
+  color: $black;
```
Comment (kaxcode) [inline ui-v2/app/styles/components/app-view/layout.scss:24]: @opihana @jnwright These are links, are they meant to be black/gray/blue when _not_ in hover?

Comment (jnwright) [reply ui-v2/app/styles/components/app-view/layout.scss:24]: Which link is $app-view-header referencing?

Comment (kaxcode) [reply ui-v2/app/styles/components/app-view/layout.scss:24]: Screen Shot 2020-04-22 at 10 17 01 AM
This one here. Below the header title, above the tabs.

Comment (jnwright) [reply ui-v2/app/styles/components/app-view/layout.scss:24]: I _think_ those should be blue and underlined.

Comment (kaxcode) [reply ui-v2/app/styles/components/app-view/layout.scss:24]: let me point this out in Figma

Comment (kaxcode) [reply ui-v2/app/styles/components/app-view/layout.scss:24]: Black it is 😄

**Pull request merged**